src/rosegold/world/window.cr

```diff
@@ -74,7 +74,7 @@ class Rosegold::Window
   end
 
   def hotbar : Array(WindowSlot)
-    slots[@inventory_start + 27...@inventory_start + 36]
+    slots[@inventory_start + 27...]
```

Comment:
In the player window, this will include the offhand slot (45), which is not intended.

Reply:
Doesn't the PlayerWindow subclass handle that? 
Currently this implementation doesn't work for anything that is not the player window (I imagine because @inventory_start + 36 = 0)

IMO eventually it would make sense to have a dedicated window class for every type listed here https://wiki.vg/Inventory so you can't end up with weird offset errors like this.

Reply:
I see, yeah you're right this method is indeed now overridden in PlayerWindow. I guess that was just some leftover code from earlier. In that case it can indeed be an open ended slice.

I don't see how `@inventory_start + 36 = 0` though; the original code should work with all windows since `@inventory_start` is set differently for each window type already.